<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Widget
{
	background-image: url(:/background/pics/1.png);
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_signup">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>180</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>sign up</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_account">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Account</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_account">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>15</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>60</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_signin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>180</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>sign in</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>65</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
